js/README.md

```diff
@@ -0,0 +1,28 @@
+# Frontend apps
```

Comment:
It seems strange that the link states "Simple JS App" and the title of this page is a more generic "Frontend Apps"

Reply:
Yes.